services/user.py

```diff
@@ -0,0 +1,75 @@
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_or_update_email_and_f_l_name(
```

Comment:
bad function name 

try something  like this 
`modify_user_data`, `update_user_information`